timeless/__init__.py

```diff
@@ -16,7 +16,13 @@ def create_app(config):
     """Creates a new Timeless webapp given a config class"""
     app = Flask(__name__, instance_relative_config=True)
     app.config.from_object(config)
-    cache = Cache(app, config={"CACHE_TYPE": "redis"})
+    CACHE.init_app(
```

Comment:
@ammaratef45 can we just pass `config`, like `CACHE.init_app(app, config=config)`? And define `CACHE_TYPE` and `CACHE_REDIS_HOST` in config object